src/analyzer/Manager.cc

```diff
@@ -110,7 +110,7 @@ void Manager::DumpDebug() {
 void Manager::Done() {}
 
 bool Manager::EnableAnalyzer(const zeek::Tag& tag) {
-    Component* p = Lookup(tag);
+    Component* p = Lookup(tag, false);
```

Comment:
Do we somewhere else take mappings into account when toggling an analyzer? I think there should be a way to do that (worst case: through an extra argument here and the bifs calling it). I am thinking of a use case where a user loads e.g., spicy-http (which replaces the building `HTTP`), and they should have a way to transparently specify from scriptland (e.g., `local.zeek`) that they do not want HTTP analysis without having to be aware of all possible mappings.

Similarly below.

Reply:
Toggling HTTP as a protocol is a different thing from toggling a specific HTTP analyzer. This API has always been referring to specific analyzers, i.e., here one particular implementation of HTTP, that can be toggled. That's what both the analyzer tags und names refer to. So this doesn't change anything, nor should mismatch with expectations afaict.